CONTRIBUTING.md

```diff
@@ -1,14 +1,30 @@
 # How to Contribute
 
-The Node.js SDK, as well as the rest of the Temporal codebase is open sourced under the MIT license.
+The Node.js SDK (as well as the rest of the Temporal codebase) is open sourced under the MIT license.
 
-We welcome contributions from the community. To contribute please start by opening an [issue](https://github.com/temporalio/sdk-node/issues) and dicussing the proposed change, once a change has been agreed upon development may start and submitted via a [pull request](https://github.com/temporalio/sdk-node/pulls).
+We welcome contributions from the community. To contribute please start by opening an [issue](https://github.com/temporalio/sdk-node/issues) and discussing the proposed change. Once a change has been agreed upon, development may start and be submitted via a [pull request](https://github.com/temporalio/sdk-node/pulls).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
Maybe add a little explanation or an npm script to generate this TOC?

Reply:
Added a script and also applied it to README